doc/sample-nginx.conf

```diff
@@ -0,0 +1,62 @@
+worker_processes  1;
+
+events {
```

Comment:
remove this - we should only change defaults where necessary

this keeps config simple and easy to understand - the changed things are intentional